public/app/features/explore/TimeSyncButton.test.tsx

```diff
@@ -8,10 +8,6 @@ const setup = (isSynced: boolean) => {
 };
```

Comment:
This doesn't look like it is related, but I like the change and removing of snapshot test. 💙

Reply:
Removed, as it failed in a weird way after bumping unicons. So the snapshot didn't make any sense